### PR TITLE
Nixify config.toml for hyperspace #8669dck7b

### PR DESCRIPTION
**Pull request by obsessed-cake**

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [NX-83 Nixify config.toml](https://app.clickup.com/t/20465559/NX-83)

File: inputs/centauri/hyperspace.nix [hunk 1]
```diff
@@ -1,6 +1,37 @@
 { self, ... }: {
   perSystem = { config, self', inputs', pkgs, system, crane, ... }: {
     packages = rec {
+      hyperspace-config = pkgs.writeText "config.json" ''
```
Comment (KaiserKarel) [inline inputs/centauri/hyperspace.nix:4]: You could pass in a config object; and do 

```suggestion
      hyperspace-config = pkgs.writeText (builtins.toJSON config)
```

then fields in this config.json can be easily overridden and set in nix.

Comment (obsessed-cake) [reply inputs/centauri/hyperspace.nix]: hyperspace uses config in TOML format.
my bad, I use the wrong extension of the file.

Comment (KaiserKarel) [reply inputs/centauri/hyperspace.nix]: No bultins.toToml :(

**Review by dzmitry-lahoda — APPROVED** [review]

**Comment by obsessed-cake** [comment]
This is a duplicate of https://github.com/ComposableFi/composable/pull/3033

**Pull request closed without merging**